### ISSUE TITLE
Unable to mount

### ISSUE THREAD

**Issue by topcoco** [adafruit/Adafruit_TinyUSB_Arduino#458]
### Operating System

Windows 11

### Arduino IDE version

1.8

### Board

esp32s3

### ArduinoCore version

1

### TinyUSB Library version

3.3

### Sketch as ATTACHED TXT

![image](https://github.com/user-attachments/assets/4c15c4df-7955-4597-a703-c46a98e80619)

### Compiled Log as ATTACHED TXT

![image](https://github.com/user-attachments/assets/eeb73cf9-ee38-4eb1-af95-c7bb1966103c)
#include <Adafruit_TinyUSB.h>

// 单一报告（无 ID）描述符
uint8_t const desc_hid_report[] = {
    TUD_HID_REPORT_DESC_MOUSE()
};

// USB HID 对象
Adafruit_USBD_HID usb_hid;

void setup() {
    // Manual begin() is required on core without built-in support e.g. mbed rp2040
    if (!TinyUSBDevice.isInitialized()) {
        TinyUSBDevice.begin(0);
    }
    Serial.begin(9600);
    // Set up HID
    usb_hid.setBootProtocol(HID_ITF_PROTOCOL_MOUSE);
    usb_hid.setPollInterval(2);
    usb_hid.setReportDescriptor(desc_hid_report, sizeof(desc_hid_report));
    usb_hid.setStringDescriptor("TinyUSB Mouse");
    usb_hid.begin();
    // If already enumerated, additional class driverr begin() e.g msc, hid, midi won't take effect until re-enumeration
    if (TinyUSBDevice.mounted()) {
        TinyUSBDevice.detach();
        delay(10);
        TinyUSBDevice.attach();
    }

    Serial.println("Adafruit TinyUSB HID Mouse example");
}

void process_hid() {
    // 如果 HID 准备好了
    if (usb_hid.ready()) {
        uint8_t const report_id = 0; // 无 ID
        int8_t const delta = 5; // 移动步幅
        usb_hid.mouseMove(report_id, delta, delta); // 模拟向右和向下移动
    }
}

void loop() {
#ifdef TINYUSB_NEED_POLLING_TASK
    TinyUSBDevice.task();
#endif

    if (TinyUSBDevice.mounted()) {
        Serial.println("Device mounted successfully.");
        return;
    } else {
        Serial.println("Device not mounted.");
    }
    // 每100毫秒移动一次鼠标
    static uint32_t ms = 0;
    if (millis() - ms > 100) { // 调整时间间隔来改变移动频率
        ms = millis();
        process_hid();
    }
}

### What happened ?

![image](https://github.com/user-attachments/assets/bf61dccb-0c11-4fdc-851e-e5359c3461f7)

### How to reproduce ?

![image](https://github.com/user-attachments/assets/8dd02ef6-0024-4aa2-b079-0b0e6b7547ca)
Unable to mount

### Screenshots

![image](https://github.com/user-attachments/assets/377e4c9a-2706-4136-94a6-755f3f127a94)

**Comment by hathach** [comment]
ArduinoCore is too old, tried with latest release.
